src/expr/src/vector_op/string.rs

```diff
@@ -381,3 +381,95 @@ pub fn quote_ident(s: &str, writer: &mut dyn Write) {
     }
     write!(writer, "\"").unwrap();
 }
+
+/// Returns the first n characters in the string.
+/// If n is a negative value, the function will return an empty string.
```

Comment:
Sorry for the misinformation provided in the issue. The PostgreSQL behavior on negative input is actually to return all but last |n| characters.

Reply:
I had question actually why it's not the same as PostgreSQL. Updated the code